ajax/libs/angular-scroll/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "angular-scroll",
+  "description": "Scrollspy, animated scrollTo and scroll events",
+  "author": "Joel Arvidsson",
+  "homepage": "https://github.com/oblador/angular-scroll",
+  "keywords": [
+    "angular",
+    "smooth-scroll",
+    "scrollspy",
+    "scrollTo",
+    "scrolling"
+  ],
+  "filename": "angular-scroll.min.js",
+  "version": "0.6.4",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/oblador/angular-scroll.git"
+  },
+  "bugs": {
+    "url": "https://github.com/oblador/angular-scroll/issues"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://git@github.com:oblador/angular-scroll.git",
```

Comment:
Of course the url in line 25 is in wrong format, I think you can try it with git clone like this
`git clone git://git@github.com:oblador/angular-scroll.git`
and then you'll get the error message, I think you mixed the ssh and git protocol in the same url.